data/lessons.json.unvalidated.in

```diff
@@ -264,10 +264,10 @@
                 "input": "console",
                 "mapper": ["shell", "raw_output", {
                     "type": "regex",
-                    "value": "^always-show-log-out$"
+                    "value": "^can-change-accels$"
```

Comment:
What was the reason for changing this one here? iirc in the lesson we sort the keys and `always-show-log-out` was on top. Has this been removed?